pandas/tests/series/test_operators.py

```diff
@@ -189,7 +190,21 @@ def test_scalar_na_logical_ops_corners(self):
         operator.and_,
         operator.or_,
         operator.xor,
-
+        pytest.param(ops.rand_,
+                     marks=pytest.mark.xfail(reason="GH#22092 Index "
+                                                    "implementation returns "
+                                                    "Index",
+                                             raises=AssertionError,
+                                             strict=True)),
+        pytest.param(ops.ror_,
+                     marks=pytest.mark.xfail(reason="Index.get_indexer "
+                                                    "with non unique index",
+                                             raises=InvalidIndexError,
+                                             strict=True)),
+        # pytest.param(ops.rxor,
```

Comment:
what to do about this test? It no longer fails, but the output is not too sensible.

Reply:
can you show an example of this and why its not sensible?

Reply:
Looking a little further into it, I guess it does technically make sense as the underlying method for an Index xor is symmetric_difference. However the xor behavior for Index and Series are different, and this test is tailored towards the series behavior.

I'll break up the reversed and non reversed tests.

Reply:
done.